plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPropertiesFileProvider.kt

```diff
@@ -0,0 +1,53 @@
+package io.sentry.android.gradle
+
+import com.android.build.gradle.api.ApplicationVariant
+import org.gradle.api.Project
+import java.io.File
+
+internal object SentryPropertiesFileProvider {
+
+    private const val FILENAME = "sentry.properties"
+
+    /**
+     * Find sentry.properties and returns the path to the file.
+     * @param project the given project
+     * @param variant the given variant
+     * @return A [String] for the path if sentry.properties is found or null otherwise
+     */
+    @JvmStatic
+    fun getPropertiesFilePath(project: Project, variant: ApplicationVariant): String? {
```

Comment:
let's use the File.separator API instead of `/`

Reply:
Yup updated in tests as well. Is not super elegant though (and also not really needed when dealing with Gradle API). Though for the future we should probably clean this up and use `Path` instead.